src/main/java/eu/europeana/metis/sandbox/domain/Record.java

```diff
@@ -38,6 +48,19 @@ private Record(String recordId, String datasetId, String datasetName,
    * @throws NullPointerException if any parameter is null
    */
   public static Record from(Record record, String content) {
+    requireNonNull(content);
+    return from(record, content.getBytes(DEFAULT_CHARSET));
```

Comment:
There are steps (metis libraries) that return a String as the result of the process. So we keep this one, and is preferred since it will use UTF-8 charset